tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -60,6 +60,25 @@ export class Ide {
         await this.driverHelper.waitVisibility(notificationLocator, timeout);
     }
 
+    async waitTaskExitCodeNotificationBoolean(exitCode: string, timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) : Promise<boolean> {
+        Logger.debug(`Ide.waitTaskExitCodeNotification "has exited with code ${exitCode}."`);
+
+        const exitCodeNotificationLocator: By = By.xpath(this.getNotificationXpathLocator(`has exited with code`));
+        const notificationLocator: By = By.xpath(this.getNotificationXpathLocator(`has exited with code ${exitCode}.`));
+
+        Logger.info(`Ide.waitTaskExitCodeNotification waiting for any exit code notification.`);
+        try {
+            await this.driverHelper.waitVisibility(exitCodeNotificationLocator);
+        } catch (err) {
+            if (err instanceof error.TimeoutError) {
+                Logger.error(`Ide.waitTaskExitCodeNotificationBoolean wait for notification timed out.`);
+            }
+            throw err;
+        }
+        Logger.info(`Ide.waitTaskExitCodeNotification checking for correct exit core:${exitCode}`);
+        return await this.driverHelper.waitVisibilityBoolean(notificationLocator, 1, 1000);
```

Comment:
It seems that it's not needed to define the `polling` as it defaults to `1000`.

Reply:
I wanted to be explicit

Reply:
in case the default variable changes, I still want it to only be a second for this case

Reply:
I think that using default should be better - if e.g. infra will slow down and we would need to increase the wait for visibility, it will mean we need to go through those occurrences and increase that manually everywhere - which is exactly what we want to avoid by using the default variable.